core/src/test/scala/com/avast/clients/rabbitmq/LiveTest.scala

```diff
@@ -396,19 +397,18 @@ class LiveTest extends TestBase with ScalaFutures {
 
       rabbitConnection.newStreamingConsumer[Bytes]("testingStreaming", Monitor.noOp()).withResource { cons =>
         val stream = cons.deliveryStream
-          .mapAsyncUnordered(20) { del =>
+          .mapAsyncUnordered(50) { del =>
```

Comment:
This value seems quite arbitrary. Shouldn't it configurable?

Reply:
This is usage in test, I don't see what should be configurable here... I've increased the value just to speedup the test (hopefully).